### PR TITLE
Card to Card mapping App changes

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/si/settings/employee-mappings/employee-mappings.component.html [hunk 1]
```diff
@@ -53,12 +53,12 @@
...
 
 
-      <ng-container matColumnDef="ccc">
+      <!-- <ng-container matColumnDef="ccc">
         <th mat-header-cell *matHeaderCellDef> Charge Card</th>
         <td mat-cell *matCellDef="let element">
           {{element.destination_card_account ? element.destination_card_account?.value : null}}
         </td>
-      </ng-container>
+      </ng-container> -->
```
Comment (DhaaraniCIT) [inline src/app/si/settings/employee-mappings/employee-mappings.component.html:61]: shall I remove it or leave it as it is?

Comment (ashwin1111) [reply src/app/si/settings/employee-mappings/employee-mappings.component.html:61]: remove it off

File: src/app/si/settings/sage-intacct-configurations/configuration/configuration.component.ts [hunk 2]
```diff
@@ -406,6 +406,7 @@ export class ConfigurationComponent implements OnInit {
 
     const importProjects = that.configurationForm.value.importProjects ? that.configurationForm.value.importProjects : false;
     const importTaxCodes = that.configurationForm.value.importTaxCodes ? that.configurationForm.value.importTaxCodes : false;
+    const chargeCard = that.configurationForm.value.cccExpense === 'CHARGE_CARD_TRANSACTION' ? true : false;
```
Comment (ashwin1111) [inline src/app/si/settings/sage-intacct-configurations/configuration/configuration.component.ts:409]: `isCCTExportEnabled`

File: src/app/si/settings/employee-mappings/employee-mappings.component.html [hunk 3]
```diff
@@ -53,12 +53,12 @@
       </ng-container>
 
 
-      <ng-container matColumnDef="ccc">
```
Comment (ashwin1111) [inline src/app/si/settings/employee-mappings/employee-mappings.component.html:56]: employee mapping dialog is not handled

**Review by ashwin1111 — COMMENTED** [review]
Have a green for the PR, once API is approved, will approve this PR as well

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**